demodj/settings.py

```diff
@@ -0,0 +1,111 @@
+"""
+Django settings for demodj project.
+
+Generated by 'django-admin startproject' using Django 5.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
I don't think `resolve` is necessary

Reply:
Might not be. I'll leave it since it was auto-generated by startproject and I don't see it being a problem.